cms/core/tests/test_views.py

```diff
@@ -1,14 +1,18 @@
 import logging
 
-import pytest
+from django.test import Client, TestCase
 
 
-@pytest.mark.django_db
-def test_csrf_token_mismatch_logs_an_error(csrf_check_client, caplog, enable_console_logging):  # pylint: disable=unused-argument
-    """Check that the custom csrf error view logs CSRF failures."""
-    csrf_check_client.cookies["csrftoken"] = "wrong"
+class CSRFTestCase(TestCase):
+    """Tests for CSRF enforcement."""
+    def setUp(self):
+        self.client = Client(enforce_csrf_checks=True)
```

Comment:
suggestion/nit: move to `setUpTestData`

Reply:
Django's `TestCase` creates the client for each case, so I think the same should be done here.